lib/binance_api/base.rb

```diff
@@ -88,5 +90,13 @@ def safe
     rescue RestClient::ExceptionWithResponse => err
       return err.response
     end
+
+    def setup_proxy(proxy)
+      RestClient.proxy = proxy
```

Comment:
I'd suggest setting a proxy URL for the exact instance rather than static `RestClient`. This would affect other requests while these are executed at the same time.